src/features/add-card-button/style.ts

```diff
@@ -11,7 +11,7 @@ export const buttonStyle: SxProps = {
 };
 
 export const titleStyle: SxProps = {
-  fontSize: 16,
+  fontSize: '1rem',
   fontWeight: 500,
-  lineHeight: 1.5,
+  lineHeight: 1.2,
```

Comment:
<img width="536" alt="Снимок экрана 2023-11-16 в 20 50 51" src="https://github.com/Online-discount-card-service/odcs-frontend/assets/5512536/481aa374-82a2-4794-a42f-6399309ad64d">

```suggestion
  lineHeight: 1.5,
```

Reply:
намерено тут изменила  lineHeight на 1.2, потому что так более собрано смотриться, иначе слово "карту" отваливается от группы